### PR TITLE
Added EFCore models and connection to database 

### PR DESCRIPTION
**Pull request by JonasKaad**
- Implemented models of the database according to `EFCore`.
- Added connection to the database using `DBContext`.
- Added an example page for interacting with the database.

### CODE REVIEW

File: FlightPatternDetection.DTO/Properties/launchSettings.json
Comment (AlexanderNorup) [inline FlightPatternDetection.DTO/Properties/launchSettings.json:1]: Please remove

File: FlightPatternDetection/Models/HoldingPattern.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    public string Fixpoint { get; set; }
+
+    public int Laps { get; set; }
+
+    public enum Direction
+    {
+        Right,
+        Left
+    }
```
Comment (AlexanderNorup) [inline FlightPatternDetection/Models/HoldingPattern.cs:20]: This is a definition of a new enum. Not a reference to it. 
You should move the selected to another file and then add `public Direction Direction {get; set;}` here instead.

File: FlightPatternDetection/Models/RouteInformation.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+public class RouteInformation
+{
+    [Key, ForeignKey(nameof(Flight.Flight_Id))]
+    public int Flight_Id { get; set; }
+    [ForeignKey(nameof(Airport.ICAO))]
+    public string Destination_ICAO { get; set; }
+    [ForeignKey(nameof(Airport.ICAO))]
+    public string Origin_ICAO { get; set; }
+    public DateTime Takeoff_Time { get; set; }
+    public string ATC_Route { get; set; }
```
Comment (AlexanderNorup) [inline FlightPatternDetection/Models/RouteInformation.cs:17]: No underscores in names please.

File: UnitTests/Properties/launchSettings.json
Comment (AlexanderNorup) [inline UnitTests/Properties/launchSettings.json:1]: Please remove

File: TrafficApiClient/Properties/launchSettings.json
Comment (AlexanderNorup) [inline TrafficApiClient/Properties/launchSettings.json:1]: Please remove

**Review by AlexanderNorup — CHANGES_REQUESTED** [review]

File: FlightPatternDetection.DTO/Properties/launchSettings.json
Comment (JonasKaad) [reply FlightPatternDetection.DTO/Properties/launchSettings.json:1]: Removed.

File: FlightPatternDetection/Models/HoldingPattern.cs (hunk 1 above)
Comment (JonasKaad) [reply FlightPatternDetection/Models/HoldingPattern.cs:20]: An Enum class has been created instead

File: FlightPatternDetection/Models/RouteInformation.cs (hunk 2 above)
Comment (JonasKaad) [reply FlightPatternDetection/Models/RouteInformation.cs:17]: Everything has been updated to PascalCase

File: FlightPatternDetection/ApplicationDbContext.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    {
+        ;
+    }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        base.OnModelCreating(modelBuilder);
+        modelBuilder.Entity<RouteInformation>().Navigation(e => e.Destination).AutoInclude();
+        modelBuilder.Entity<RouteInformation>().Navigation(e => e.Origin).AutoInclude();
+        //modelBuilder.Entity<HoldingPattern>().Property(p => p.Direction).HasColumnType("Direction");
```
Comment (VictorABoye) [inline FlightPatternDetection/ApplicationDbContext.cs:28]: Outdated?

**Review by VictorABoye — APPROVED** [review]
Only one place in OnModelCreating that I think was overlooked

File: FlightPatternDetection/ApplicationDbContext.cs (hunk 3 above)
Comment (JonasKaad) [reply FlightPatternDetection/ApplicationDbContext.cs:28]: Outdated.

**Pull request merged**